jdaviz/configs/specviz/plugins/unit_conversion/tests/test_unit_conversion.py

```diff
@@ -206,17 +206,39 @@ def test_sb_unit_conversion(cubeviz_helper):
     uc_plg.sb_unit = 'Jy / sr'
     y_display_unit = u.Unit(viewer_1d.state.y_display_unit)
     assert y_display_unit == u.Jy / u.sr
+    label_mouseover = cubeviz_helper.app.session.application._tools['g-coords-info']
+    flux_viewer = cubeviz_helper.app.get_viewer(cubeviz_helper._default_flux_viewer_reference_name)
+    label_mouseover._viewer_mouse_event(flux_viewer, {'event': 'mousemove',
+                                                      'domain': {'x': 1, 'y': 1}})
+    assert label_mouseover.as_text() == (
+            "Pixel x=00001.0 y=00001.0 Value +0.00000e+00 Jy / sr",
+            "World 13h39m59.9461s +27d00m00.7200s (ICRS)",
+            "204.9997755344 27.0001999998 (deg)")
 
     # Try a second conversion
     uc_plg.sb_unit = 'W / Hz sr m2'
     y_display_unit = u.Unit(viewer_1d.state.y_display_unit)
     assert y_display_unit == u.Unit("W / (Hz sr m2)")
 
+    y_display_unit = u.Unit(viewer_1d.state.y_display_unit)
+    label_mouseover._viewer_mouse_event(flux_viewer, {'event': 'mousemove',
+                                                      'domain': {'x': 1, 'y': 1}})
+    assert label_mouseover.as_text() == (
+            "Pixel x=00001.0 y=00001.0 Value +0.00000e+00 W / (Hz sr m2)",
```

Comment:
Maybe you want to pick a pixel position where the flux isn't zero, to make sure that the conversion is actually correct.

Also maybe try a unit (or two) that would need special equivalencies (per-freq to per-wave, or without sr).